DesktopBridge.Helpers/Helpers.cs

```diff
@@ -1,15 +1,14 @@
 ﻿using System;
 using System.Runtime.InteropServices;
-using System.Text;
 
 namespace DesktopBridge
 {
-    public class Helpers
+    public unsafe class Helpers
     {
         const long APPMODEL_ERROR_NO_PACKAGE = 15700L;
 
-        [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
-        static extern int GetCurrentPackageFullName(ref int packageFullNameLength, StringBuilder packageFullName);
+        [DllImport("kernel32.dll", ExactSpelling = true)]
+        static extern int GetCurrentPackageFullName(uint* packageFullNameLength, char* packageFullName);
```

Comment:
[Documentation says](https://docs.microsoft.com/en-us/windows/win32/api/appmodel/nf-appmodel-getcurrentpackagefullname) the first parameter is an unsigned integer.